docs/install/install_options/install_options.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Options
```

Comment:
Nit:
(I missed this in my original) Can we just call it "Overview" instead of "Options" because in the nav tree this looks funny to me:
```
Install Options:
        Options
```